app/models/library_entry.rb

```diff
@@ -48,6 +48,7 @@ class LibraryEntry < ApplicationRecord
   belongs_to :drama
   has_one :review, dependent: :destroy
   has_many :library_events, dependent: :destroy
+  has_many :media_reactions
```

Comment:
I was unsure, since `has_one` from the docks says "This association indicates that each instance of a model contains or possesses one instance of another model.", which isn't true because library entries don't have to have any media reactions. 

Compared to `has_many` which implies: "This association indicates that each instance of the model has zero or more instances of another model." which I thought would be more permissive.